locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -2960,42 +2868,36 @@ msgid "Not set"
 msgstr "Non défini"
 
 #: bookwyrm/templates/user_admin/user_info.html:5
-#, fuzzy
-#| msgid "Details"
 msgid "User details"
-msgstr "Détails"
+msgstr "Détails du compte"
 
 #: bookwyrm/templates/user_admin/user_info.html:14
 msgid "View user profile"
 msgstr "Voir le profil"
 
 #: bookwyrm/templates/user_admin/user_info.html:20
-#, fuzzy
-#| msgid "Instance Settings"
 msgid "Instance details"
-msgstr "Paramètres de l’instance"
+msgstr "Détails de l’instance"
 
 #: bookwyrm/templates/user_admin/user_info.html:46
 msgid "View instance"
-msgstr ""
+msgstr "Voir l’instance"
 
 #: bookwyrm/templates/user_admin/user_moderation_actions.html:11
 msgid "Suspend user"
-msgstr ""
+msgstr "Suspendre le compte"
 
 #: bookwyrm/templates/user_admin/user_moderation_actions.html:13
 msgid "Un-suspend user"
-msgstr ""
+msgstr "Rétablir le compte"
 
 #: bookwyrm/templates/user_admin/user_moderation_actions.html:21
 msgid "Access level:"
-msgstr ""
+msgstr "Niveau d’accès :"
 
 #: bookwyrm/views/import_data.py:67
-#, fuzzy
-#| msgid "Email address:"
 msgid "Not a valid csv file"
-msgstr "Adresse email :"
+msgstr "Fichier CSV invalide"
```

Comment:
? I have no stats to back up the following but I would have gone for _non valide_ instead of _invalide_ which I feel is more used in the sense of _disabled people_.